bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -24,8 +24,10 @@
   {{ _('Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource, open and accessible to all.') }}
 {% endblock %}
 
+{% set cop = request.GET.get('v', None) %}
```

Comment:
We generally prefer to send this via a context variable rather than accessing `request.GET` directly in the template. There's a basic example [here](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L103).

It's also a good idea to ensure the variable grabbed from the query string is one of a pre-defined set of acceptable values (so someone can't just hack the URL and inject whatever).

In the view, something like:

```
cop = request.GET.get('v', None)
if cop not in ['a', 'b']:
    cop = None

ctx = {'cop': cop}
return l10n_utils.render(request, 'mozorg/home/home.html', ctx)
```

The above is all assuming that there will be a Traffic Cop experiment added to this PR.

Reply:
It's coming out but apparently I missed some bits. Thanks :)